lib/modules/platform/github/index.ts

```diff
@@ -1950,6 +1950,7 @@ export function massageMarkdown(input: string): string {
     .replace(regEx(/]: https:\/\/github\.com\//g), ']: https://togithub.com/')
     .replace('> ℹ **Note**\n> \n', '> [!NOTE]\n')
```

Comment:
is there also a colorful info emoji?

Reply:
Yes, but I guess we don't care about other. I see only warning one behaving weird.